vendor/bat-native-ads/src/bat/ads/internal/ads/serving/choose/sample_ads.h

```diff
@@ -56,6 +68,19 @@ absl::optional<T> SampleAdFromPredictors(
   return absl::nullopt;
 }
 
+inline std::vector<double> ComputeProbabilities(std::vector<int> scores) {
```

Comment:
Should we return early if scores is empty? Can scores ever be empty?

Reply:
Added a unit test 👍 We could return early, should we return zero? Although it would be kind of unexpected to pass an empty array to ComputeProbabilities

Reply:
What scenarios could the vector be empty?

Reply:
only when `paced_creative_ads` is empty, but we should probably early return in that case rather than trying to predict on an empty set, right?